### ISSUE TITLE
MacOS: ERR_CONTENT_LENGTH_MISMATCH for webpack modules when debugging in dotnet core 2.0

### ISSUE THREAD

**Issue by bcanzanella** [OmniSharp/omnisharp-vscode#1763]
## Environment data
`dotnet --info` output:
```
.NET Command Line Tools (2.0.0)

Product Information:
 Version:            2.0.0
 Commit SHA-1 hash:  cdcd1928c9

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.12
 OS Platform: Darwin
 RID:         osx.10.12-x64
 Base Path:   /usr/local/share/dotnet/sdk/2.0.0/

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0
  Build    : e8b8861ac7faf042c87a5c2f9f2d04c98b69f28d
  ```
VS Code version: 1.17-insiders
C# Extension version: 1.13.0-beta2

## Steps to reproduce

I've created an example repo that shows the issue. This only happens when attached to the debugger, `dotnet run` works as expected. There's a vscode launch configuration called 'Web' to start debugging. I've tested this on MacOS, Ubuntu (Kubuntu) and Windows 10 (see Actual behaviors below).

```
git clone https://github.com/ioninteractive/omnisharp-webpack-hmr-issue

dotnet restore

dotnet build

npm install
```
vscode -> launch -> 'Web'

visit http://localhost:5000/dist/index.html

Check the responses for app.[hash].js and vendor.[hash].js

## Expected  behavior
webpack hot module JS files (app.[hash].js and vendor.[hash].js) return sub-second `200` responses.

## Actual behavior
MacOS: JS hot modules attempt to load (but buffer instead) until a `ERR_CONTENT_LENGTH_MISMATCH` is returned.
Kubuntu 16.04: JS hot modules buffer, then eventually load, taking between 5-10 seconds per file.
Windows 10: JS hot modules load with sub-second `200` responses.

**Comment by gregg-miskelly** [comment]
@bcanzanella This doesn't sound like it is a debugging issue. Do you see the same results when you just start the process on the console?

**Comment by bcanzanella** [comment]
it works flawlessly with `dotnet run` across all OSes.

**Comment by gregg-miskelly** [comment]
Did you try it with `ASPNETCORE_ENVIRONMENT=Development` set?

**Comment by bcanzanella** [comment]
yes, it's set as an environment variable `export ASPNETCORE_ENVIRONMENT="Development"` (FWIW, it was also set as part of the launched task too)

**Comment by eamodio** [comment]
This also only started to happen after the upgrade to dotnet core 2.0 -- we saw no issues (with the debugger attached) in 1.1.

**Comment by gregg-miskelly** [comment]
@davidfowl Do you have any ideas what might be causing this?

**Comment by davidfowl** [comment]
@bcanzanella can you share a repository?

/cc @SteveSanderson

**Comment by bcanzanella** [comment]
@davidfowl sure thing! https://github.com/ioninteractive/omnisharp-webpack-hmr-issue

**Comment by bcanzanella** [comment]
Is there anything else I can provide to help resolve this? This is a rather nasty issue for us and affects our entire team; adding seconds to every webpack hot-reloadable module request when debugging.

I also noticed that this is tagged as a question, but it really does seem like a bug introduced in 2.0...
